assets/stylesheets/sections/gutenberg-editor.scss

```diff
@@ -9,15 +9,17 @@
 
 // Block library
 @import '../../../node_modules/@wordpress/block-library/build-style/style';
-@import '../../../node_modules/@wordpress/block-library/build-style/editor';
-@import '../../../node_modules/@wordpress/block-library/build-style/theme';
 
 // Components
 @import '../../../node_modules/@wordpress/components/build-style/style';
 
 // Editor package styles
 @import '../../../node_modules/@wordpress/editor/build-style/style';
 
+// Block library theme & editor
```

Comment:
Have you by any chance tried moving here the remaining block library import (line 11)?
`@import '../../../node_modules/@wordpress/block-library/build-style/style';`

It could help us even more, and anyway it would be in the same position as in core. 🙂

Reply:
I was tempted to do it, but I realized that Core was importing the CSS files in this order:

<img width="1488" alt="screen shot 2018-10-26 at 18 03 27" src="https://user-images.githubusercontent.com/1233880/47578395-91fe4400-d949-11e8-8e05-e46e3317f657.png">

Although I'm not sure how [these dependencies](https://github.com/WordPress/gutenberg/blob/0a1d6e0ea99f331162f48907ab93ed9b8062b634/lib/client-assets.php#L757-L839) are transformed to the CSS imports above.

I have just made a quick test moving `@import '../../../node_modules/@wordpress/block-library/build-style/style'` just before `@import '../../../node_modules/@wordpress/block-library/build-style/theme';` and I don't appreciate any change, but I think we should maintain the same order as Core:
1. block-library/style
2. components
3. ~nux~
4. editor
5. block-library/theme
6. block-library/editor
7. edit-post

Reply:
You're right. I think I got confused while reading the file.
I'd love to keep the same order as Core, and your version is the correct one at doing so. 👍